tests/integration/update_cluster/complex/in-v1alpha2.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: kops.k8s.io/v1alpha2
+apiVersion: kops/v1alpha2
```

Comment:
Please revert all changes here, except the ones needed for the feature.

Reply:
Wil address all of your review comments! Thank you for your patience and time @hakman !

Reply:
Thanks for your patience and for doing this. Will be quite a cool new feature. :)